e2e/history_components.test.js

```diff
@@ -0,0 +1,29 @@
+const { log } = require('detox');
+
+describe('Example', () => {
+  beforeAll(async () => {
+    await device.launchApp();
+  });
+
+  beforeEach(async () => {
+    // jest.useFakeTimers();
```

Comment:
How about replace this comment with:

`// Before each time the react native app is loaded, the *it* clauses are evaluated in order (if they ever occur out of order the test fails) the test will fail`